### PR TITLE
Image: lazy load in el-carousel

### PR DESCRIPTION
**Pull request by violetice**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

issue #15381
Docs: Other language no translation only simplified Chinese

### CODE REVIEW

**Comment by violetice** [comment]
@ziyoung 我这个怎么整

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[*]
```
Comment (island205) [inline .editorconfig]: 这个文件不要提交上来

**Comment by island205** [comment]
@violetice Please resolve the conflicts first.

**Comment by island205** [comment]
I'll close this PR since it's been a while since it's been opened. Feel free to reopen if you have updates on this

**Pull request closed without merging**